### PR TITLE
Feature/header

### PR DESCRIPTION
**Pull request by ngocphong230801**
In this pull request : 
- Add code for header of index file.
- Add CSS header file.

### CODE REVIEW

File: practices/practice-one/src/index.html [hunk 1]
```diff
@@ -3,10 +3,36 @@
...
         <meta http-equiv="X-UA-Compatible" content="IE=edge">
-        <meta name="viewport" content="width=device-width, initial-scale=1.0">
-        <title>Document</title>
+        <title>Practice one</title>
+        <link rel="stylesheet" href="../src/styles/style.css">
     </head>
     <body>
-        <h1>hello</h1>
+        <div class="container">
+              <!-- Start Header -->
```
Comment (cunguyen-agilityio) [inline practices/practice-one/src/index.html]: Header is enough

File: practices/practice-one/src/index.html [hunk 2]
```diff
@@ -3,10 +3,36 @@
...
+           <header>
+            <nav class="navbar">
+                <div class="logo">
+                    <img src="../src/assets/img/LogoSmallSolid.png" alt="logo" class="logo-img">
```
Comment (cunguyen-agilityio) [inline practices/practice-one/src/index.html]: ali= "VanGoux"

File: practices/practice-one/src/index.html [hunk 3]
```diff
@@ -3,10 +3,36 @@
...
+                    <a href="" class="title-logo">VanGoux</a>
+                </div>
+                <div class="menu">
+                    <div class="menu-item-header">
+                        <a href=""  class="menu-content">How it works</a>
```
Comment (cunguyen-agilityio) [inline practices/practice-one/src/index.html]: Space, href="#" -> replace for all

File: practices/practice-one/src/index.html [hunk 4]
```diff
@@ -3,10 +3,36 @@
...
+                    </div>
+                    <div class="menu-item-header">
```
Comment (cunguyen-agilityio) [inline practices/practice-one/src/index.html]: menu-item-header -> menu-item

File: practices/practice-one/src/index.html [hunk 5]
```diff
@@ -3,10 +3,36 @@
...
+                        <a href=""  class="menu-content">Features</a>
+                    </div>
+                    <div class="menu-item-header">
+                        <a href=""  class="menu-content">Use cases</a>
+                    </div>
+                    <div class="menu-item-header">
+                        <a href="" class="menu-content">Schedule a demo</a>
+                    </div>
+                <button type="button" class="btn-default">Upload sketch</button>
```
Comment (cunguyen-agilityio) [inline practices/practice-one/src/index.html:24]: Code styles guide

File: practices/practice-one/src/styles/layout/header.css [hunk 6]
```diff
@@ -0,0 +1,47 @@
+.navbar {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    position: relative;
+}
+
+.logo .logo-img {
```
Comment (cunguyen-agilityio) [inline practices/practice-one/src/styles/layout/header.css]: .brand brand-logo

File: practices/practice-one/src/styles/layout/header.css [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+    position: absolute;
+    left: 0%;
```
Comment (cunguyen-agilityio) [inline practices/practice-one/src/styles/layout/header.css]: 0% -> 0

File: practices/practice-one/src/index.html [hunk 8]
```diff
@@ -2,11 +2,37 @@
...
-        <h1>hello</h1>
+        <div class="container">
+              <!--Header--> 
+           <header>
+            <nav class="navbar">
+                <div class="brand">
+                    <img src="../src/assets/img/LogoSmallSolid.png" alt="VanGoux" class="brand-logo">
+                    <a href="" class="title-logo">VanGoux</a>
+                </div>
+                <div class="menu">
```
Comment (huyenpham-agilityio) [inline practices/practice-one/src/index.html]: should use ul li tags

**Review by huyenpham-agilityio — CHANGES_REQUESTED** [review]

**Review by huyenpham-agilityio — APPROVED** [review]

**Pull request merged**